<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" tests="20" skipped="1" failures="0" errors="0" timestamp="2021-04-16T01:52:51" hostname="DESKTOP-3R4DVF7" time="0.023">
  <properties/>
  <testcase name="testUpdateWithTypedParameters" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.006"/>
  <testcase name="testQueryWithRowMapper" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.001"/>
  <testcase name="testQueryWithResultSetExtractorNoParameters" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.001"/>
  <testcase name="testExecute" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.001"/>
  <testcase name="testQueryWithRowMapperNoParameters" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.001"/>
  <testcase name="testTemplateConfiguration" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.0"/>
  <testcase name="testNullDataSourceProvidedToCtor" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.0"/>
  <testcase name="testQueryWithRowCallbackHandler" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.001"/>
  <testcase name="testExecuteArray" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.0">
    <skipped/>
  </testcase>
  <testcase name="testExecuteNoParameters" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.001"/>
  <testcase name="testQueryWithRowCallbackHandlerNoParameters" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.0"/>
  <testcase name="testExecuteWithTypedParameters" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.0"/>
  <testcase name="testQueryForObjectWithRowMapper" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.001"/>
  <testcase name="testBatchUpdateWithSqlParameterSourcePlusTypeInfo" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.002"/>
  <testcase name="testQueryWithResultSetExtractor" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.001"/>
  <testcase name="testNullJdbcTemplateProvidedToCtor" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.0"/>
  <testcase name="testBatchUpdateWithEmptyMap" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.002"/>
  <testcase name="testBatchUpdateWithSqlParameterSource" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.0"/>
  <testcase name="testUpdate" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.001"/>
  <testcase name="testBatchUpdateWithPlainMap" classname="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests" time="0.001"/>
  <system-out><![CDATA[09:52:51.713 [Test worker] WARN  org.springframework.jdbc.support.SQLErrorCodesFactory - Error while extracting database name - falling back to empty error codes
org.springframework.jdbc.support.MetaDataAccessException: DatabaseMetaData returned by Connection [Mock for Connection, hashCode: 1397376293] was null
	at org.springframework.jdbc.support.JdbcUtils.extractDatabaseMetaData(JdbcUtils.java:341) ~[main/:?]
	at org.springframework.jdbc.support.JdbcUtils.extractDatabaseMetaData(JdbcUtils.java:374) ~[main/:?]
	at org.springframework.jdbc.support.SQLErrorCodesFactory.getErrorCodes(SQLErrorCodesFactory.java:214) [main/:?]
	at org.springframework.jdbc.support.SQLErrorCodeSQLExceptionTranslator.setDataSource(SQLErrorCodeSQLExceptionTranslator.java:136) [main/:?]
	at org.springframework.jdbc.support.SQLErrorCodeSQLExceptionTranslator.<init>(SQLErrorCodeSQLExceptionTranslator.java:99) [main/:?]
	at org.springframework.jdbc.support.JdbcAccessor.getExceptionTranslator(JdbcAccessor.java:122) [main/:?]
	at org.springframework.jdbc.support.JdbcAccessor.afterPropertiesSet(JdbcAccessor.java:163) [main/:?]
	at org.springframework.jdbc.core.JdbcTemplate.<init>(JdbcTemplate.java:178) [main/:?]
	at org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplateTests.testBatchUpdateWithEmptyMap(NamedParameterJdbcTemplateTests.java:440) [test/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_271]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_271]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_271]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_271]
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50) [junit-4.12.jar:4.12]
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12) [junit-4.12.jar:4.12]
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47) [junit-4.12.jar:4.12]
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17) [junit-4.12.jar:4.12]
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26) [junit-4.12.jar:4.12]
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:239) [junit-4.12.jar:4.12]
	at org.junit.rules.RunRules.evaluate(RunRules.java:20) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325) [junit-4.12.jar:4.12]
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78) [junit-4.12.jar:4.12]
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363) [junit-4.12.jar:4.12]
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114) [gradle-testing-jvm-4.4.1.jar:4.4.1]
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57) [gradle-testing-jvm-4.4.1.jar:4.4.1]
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66) [gradle-testing-jvm-4.4.1.jar:4.4.1]
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51) [gradle-testing-base-4.4.1.jar:4.4.1]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_271]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_271]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_271]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_271]
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35) [gradle-messaging-4.4.1.jar:4.4.1]
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24) [gradle-messaging-4.4.1.jar:4.4.1]
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32) [gradle-messaging-4.4.1.jar:4.4.1]
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93) [gradle-messaging-4.4.1.jar:4.4.1]
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source) [?:?]
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108) [gradle-testing-base-4.4.1.jar:4.4.1]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_271]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_271]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_271]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_271]
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35) [gradle-messaging-4.4.1.jar:4.4.1]
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24) [gradle-messaging-4.4.1.jar:4.4.1]
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146) [gradle-messaging-4.4.1.jar:4.4.1]
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128) [gradle-messaging-4.4.1.jar:4.4.1]
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404) [gradle-messaging-4.4.1.jar:4.4.1]
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63) [gradle-base-services-4.4.1.jar:4.4.1]
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46) [gradle-base-services-4.4.1.jar:4.4.1]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_271]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_271]
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55) [gradle-base-services-4.4.1.jar:4.4.1]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_271]
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
